### PR TITLE
Jseldess/start output

### LINKED ISSUE #90: cache size options
Document the `cache-size` field at the store level.

### PR DESCRIPTION
**Pull request by jseldess**
Updated `start-a-node.md` with details about standard output (fixes https://github.com/cockroachdb/cockroach/pull/4448). @petermattis, please take a look.

Added `cache` flag to `start-a-node.md` (fixes #90). @BramGruneir, please take a look.

Also added `--http-port` flag to `start-a-node.md` and `stop-a-node.md`. 

Will update cert options in next PR.

### CODE REVIEW

File: start-a-node.md [hunk 1]
```diff
@@ -24,12 +24,14 @@ The `start` command supports the following flags, as well as [logging flags](coc
 Flag | Description
 -----|-----------
 `--attrs` | Node-level attributes specifying topography or machine capabilities. Topography might include datacenter designation (e.g., `us-west-1a`, `us-west-1b`, `us-east-1c`). Machine capabilities might include specialized hardware or number of cores (e.g., `gpu`, `x16c`). <br><br>Attributes can be arbitrary strings separated by colons. The relative geographic proximity of two nodes is inferred from the common prefix of the attributes list, so topographic attributes should be specified first and in the same order for all nodes, for example: <br><br> `--attrs=us-west-1b:gpu`
-`--certs` | The path to the directory containing the node's [security certificates](create-security-certificates.html). When starting the node with security (i.e., without the `--insecure` flag), the `--certs` flag is required. <br><br> **Default:** certs 
+`--cache` | The total size for caches, shared evenly if there are multiple storage devices. This can be in any bytes-based unit, for example: <br><br>`--cache=1000000000  -> 1000000000 bytes`<br>`--cache=1GB         -> 1000000000 bytes`<br>`--cache=1GiB        -> 0.93132257461548 bytes`
```
Comment (bdarnell) [inline start-a-node.md]: The last example in this line is incorrect. 1 GiB is 1073741824 bytes. The second example is technically correct, but I thought we had decided to treat GB and GiB as synonymous (both meaning GiB). In that case I wouldn't show both examples.

File: start-a-node.md [hunk 2]
```diff
@@ -41,6 +43,26 @@ Field | Description
...
+store[0]:  path=cockroach-data
+~~~
+
+Field | Description
+------|------------
+`build` | The version of CockroachDB you are running.
+`admin` | The URL for accessing the Admin UI.
+`sql` | The connection URL for your client.
+`logs` | The directory containing log data.
+`store[n]` | The directory containing store data, where `[n]` is the number of stores.
```
Comment (bdarnell) [inline start-a-node.md]: `[n]` is the index of the store in that directory. In this example there is one store but it displays as `store[0]`. If there were two stores there would be two lines for `store[0]` and `store[1]`.

**Comment by bdarnell** [comment]
LGTM

File: start-a-node.md (hunk 1 above)
Comment (jseldess) [inline start-a-node.md]: I THINK that's what was decided in [issue 4735](https://github.com/cockroachdb/cockroach/issues/4735), but I'm not sure. If it's ok with you, I'll just correct the GiB value for now (thanks for catching that) and then update again once the GB/GiB change has been implemented.

File: start-a-node.md (hunk 2 above)
Comment (jseldess) [inline start-a-node.md]: Thanks for clarifying this, @bdarnell. Fixing with the next commit.

File: start-a-node.md [hunk 3]
```diff
@@ -41,6 +43,26 @@ Field | Description
...
+logs:      cockroach-data/logs
+store[0]:  path=cockroach-data
+~~~
+
+Field | Description
+------|------------
+`build` | The version of CockroachDB you are running.
+`admin` | The URL for accessing the Admin UI.
+`sql` | The connection URL for your client.
+`logs` | The directory containing log data.
```
Comment (petermattis) [inline start-a-node.md]: Perhaps clarify that these are debug logs: `containing debug log data`.

**Comment by petermattis** [comment]
LGTM

File: start-a-node.md (hunk 3 above)
Comment (jseldess) [inline start-a-node.md]: Fixed with last commit. Thanks, @petermattis

**Pull request merged**